src/ViewModel/Button.php

```diff
@@ -103,6 +106,23 @@ public static function link(
         return $button;
     }
 
+    public static function speechBubble(
+        string $text,
+        bool $isActive = true,
+        string $name = null,
+        string $id = null,
+        bool $isPopulated = false
+    ) : Button {
+        $button = new static($text, self::SIZE_CUSTOM, self::STYLE_SPEECH_BUBBLE, $isActive, $name, $id, false);
+        $button->type = self::TYPE_BUTTON;
```

Comment:
strange that the constructor doesn't set `$this->type`, any reason for the difference wrt `$name` and `$id`?

Reply:
`speechBubble` is reusing the convention already established with e.g. `form`, see https://github.com/elifesciences/patterns-php/pull/524/files/8a41c5346813bb7d5e4585a2fac9dabe441ff09c#diff-151b077966c70a0674bf1be5c891636dL72